newyork.html

```diff
@@ -98,66 +109,137 @@ <h6>March 17, 2014</h6>
            
 
-            <h3>‘As Big as It Gets’</h3>
+            <h3 class="main-article">‘As Big as It Gets’</h3>
 
-            <p class="paragraph">
+            <p class="main-article">
                 In our own universe, it would serve as a window into the forces operating at energies forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s theory of general relativity, should ruffle space-time.</p>
 
-                <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+                <p class="main-article">Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
 
-                <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
+                <p class="main-article">He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
 
 
 
-                <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+                <p class="main-article">The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
 
-                <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+                <p class="main-article">Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
 
 
         
-        </div>
+        <div class="theory-container">
+        <!-- the grip photo with words -->
+           <div class="theory-words">
+            <h2>The Theory of Inflation</h2>
+            <p class="h6-theory">Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+          </div>
```

Comment:
## Possible word-wrap?
- [ ] Would you mind adding a word wrap to this section of the `theory of inflation` to resemble the original?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|![theory-of-inflation-word-wrap](https://user-images.githubusercontent.com/22345961/108571937-447c2100-7322-11eb-8420-34a9cafbccdd.png)|![theory-of-inflation-word-wrap-clone](https://user-images.githubusercontent.com/22345961/108571947-4940d500-7322-11eb-8167-02c901e7f05f.png)|